docker_build_sub/Tiltfile

```diff
@@ -41,4 +41,5 @@ def docker_build_sub(ref, context, extra_cmds, child_context=None, base_suffix='
   base_ref = '%s-base' % ref
   docker_build(base_ref, context, **kwargs)
   df = '\n'.join(['FROM %s' % base_ref] + extra_cmds)
-  docker_build(ref, child_context, dockerfile_contents=df, live_update=live_update, **kwargs)
+  child_kwargs = {k: v for k, v in kwargs.items() if k not in ["dockerfile_contents", "dockerfile", "live_update"]}
+  docker_build(ref, child_context, dockerfile_contents=df, live_update=live_update, **child_kwargs)
```

Comment:
You should remove `live_update` from the function's arguments or it won't be in `kwargs`

Reply:
good point!